src/radical/pilot/raptor/master.py

```diff
@@ -441,6 +437,8 @@ def submit_workers(self, descriptions: List[TaskDescription]
 
         self.advance(tasks, publish=True, push=True)
 
+        # dump registry with all worker descriptions ("raptor.<worker_uid>.cfg")
+        self._reg.dump('raptor_master')
```

Comment:
Is this left intentionally or was that just for debugging?  If intentional, then we should add the master UID to the file name, otherwise dumps for multiple masters could collide.

Reply:
this one was instead of other dumps we had

> we should add the master UID to the file name, otherwise dumps for multiple masters could collide

agree, will update accordingly

Reply:
done